go/coordinator/Dockerfile.migration

```diff
@@ -0,0 +1,4 @@
+FROM arigaio/atlas:latest
```

Comment:
whats a Dockerfile.migration

Reply:
just a different Dockerfile, this is for database migration for postgres